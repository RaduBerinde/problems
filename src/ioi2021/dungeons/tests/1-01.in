b50747e9-747c-4fca-b3b0-62317b32d2f6
1 73
9829
6
1
0
0 2
0 7
0 2
0 2
0 6
0 2
0 6
0 2
0 7
0 7
0 10
0 1
0 9
0 5
0 5
0 7
0 5
0 9
0 3
0 8
0 9
0 8
0 6
0 4
0 1
0 9
0 8
0 10
0 10
0 1
0 8
0 8
0 8
0 7
0 3
0 10
0 4
0 2
0 9
0 4
0 1
0 3
0 6
0 10
0 10
0 10
0 1
0 1
0 10
0 1
0 5
0 9
0 2
0 6
0 8
0 9
0 6
0 6
0 6
0 2
0 6
0 10
0 4
0 8
0 8
0 3
0 10
0 9
0 7
0 7
0 6
0 8
0 7
