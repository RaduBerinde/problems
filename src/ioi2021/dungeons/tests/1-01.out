f39eb8f7-7d10-4b4a-af02-d7aef3d4dd0a
OK
19659
19658
19659
19659
19663
19659
19663
19659
19658
19658
19661
19658
19660
19662
19662
19658
19662
19660
19660
19659
19660
19659
19663
19661
19658
19660
19659
19661
19661
19658
19659
19659
19659
19658
19660
19661
19661
19659
19660
19661
19658
19660
19663
19661
19661
19661
19658
19658
19661
19658
19662
19660
19659
19663
19659
19660
19663
19663
19663
19659
19663
19661
19661
19659
19659
19660
19661
19660
19658
19658
19663
19659
19658
